packages/@aws-cdk/aws-rds/lib/engine.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * A common interface for database engines.
+ * It has 2 sub-interfaces, IClusterEngine and IInstanceEngine.
+ * Don't implement this interface directly,
+ * instead implement one of the 2 sub-interfaces.
+ */
+export interface IEngine {
+  /** The type of the engine, for example "mysql". */
+  readonly engineType: string;
+
+  /**
+   * The exact version of the engine that is used,
+   * for example "5.1.42".
+   *
+   * @default - use the default version for this engine type
```

Comment:
when using IClusterEngine or IInstanceEngine, users would see this text for the default right?
i.e. each implementing interface couldn't communicate it's own help text?

Reply:
That's not a given. You can always add JSDocs for this property in the class that implements this interface, and it should override the displayed documentation.

Reply:
i know it'll work in typescript, but does jsii allow that (overriding a property in an extended interface)?

Reply:
It does, yes.